mllib/src/test/scala/org/apache/spark/mllib/linalg/distributed/RowMatrixSuite.scala

```diff
@@ -101,6 +101,14 @@ class RowMatrixSuite extends SparkFunSuite with MLlibTestSparkContext {
     }
   }
 
+  test("getTreeAggregateIdealDepth") {
+    val rowMat = new RowMatrix(sc.emptyRDD[Vector])
+    val driverResultSizeInMb = 10000
+    val nbPartitions = 10000
+    assert(rowMat.getTreeAggregateIdealDepth(99, nbPartitions, driverResultSizeInMb) == 2)
```

Comment:
Nit: use `===` to trigger the scalatest macro that will print a better error if this check fails.

Reply:
nice, another trick I'll apply in my code from now on, thanks.